imports/api/hmisApi/hmisClient.js

```diff
@@ -107,10 +108,20 @@ export class HmisClient {
   }
 
   static create(userId) {
+    globalLogger.warn(`
+      creating HMIS client via create() is deprecated.
+      Use method/publication context instead
+    `);
     const serviceConfig = ServiceConfiguration.configurations.findOne({ service: 'HMIS' });
     if (!serviceConfig.appId || !serviceConfig.appSecret) {
       throw new ServiceConfiguration.ConfigError();
     }
-    return new HmisClient(userId, serviceConfig, HmisApiRegistry, Meteor.users);
+    return new HmisClient({
+      userId,
+      serviceConfig,
+      hmisApiRegistry: HmisApiRegistry,
+      usersCollection: Meteor.users,
+      logger: globalLogger,
```

Comment:
shouldn't it be `this.logger`?

Reply:
it's a static method for creating loggers, that's why it is using globalLogger for backward comp.